detection-rules/link_russian_traversed_credential_phishing.yml

```diff
@@ -0,0 +1,53 @@
+name: "Link: Credential Phishing via messages with Russian TLD traversal"
+description: "This rule detects credential phishing attempts in emails traversing Russian TLDs by aggressively analyzing links for signs of phishing, including suspicious keywords, login prompts, or links flagged for credential theft, excluding emails from trusted domains unless they fail DMARC verification."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(headers.domains, .tld in ("ru", "su"))
+  and any(ml.nlu_classifier(body.current_thread.text).intents,
+          .name == "cred_theft" and .confidence in ("medium", "high")
+  )
+  and 0 < length(body.links) < 5
+  and any(body.links,
+          beta.linkanalysis(., mode="aggressive").credphish.disposition == "phishing"
+          or (
+            strings.icontains(beta.linkanalysis(., mode="aggressive").final_dom.raw,
+                              "Pàsswórd"
```

Comment:
We might want more permutations here (perhaps with `replace_confusables`?)

Reply:
For sure since it wasn't accessible globally yet I didn't include it

Reply:
Oh that's not GA... hm.